android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -711,6 +711,7 @@ private void showClearTextDialog() {
       @Override
       public void onClick(DialogInterface dialog, int which) {
         textView.setText("");
+        KMManager.resetContext(KeyboardType.KEYBOARD_TYPE_INAPP);
```

Comment:
I wonder why this is needed. I would expect that changes to the text view (such as `textView.setText()`) should be setting the context already, which suggests an issue lower down in Keyman Engine?

Reply:
Per standup, I'll put this back to Draft and we'll see if this is still needed after @jahorton 's Corrections work stabilizes